### DISCUSSION TITLE
License not work

### DISCUSSION THREAD

**Discussion by AM-Login** [aclap-dev/video-downloadhelper#441] [License & Purchase requests]
### What's your issue?

I purchased a Video DownloadHelper license through the Chrome Web Store (Lifetime Premium VDH) and now I can’t activate

It is also not possible to restore the license. I bought it about 4 years ago

### The 10 first digits (only 10) of your license number or email address

am.workpage@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
I sent you an email. 
jerome
